### ISSUE TITLE
Password

### ISSUE THREAD

**Issue by jodumont** [frappe/erpnext#9385]
Hi;

I'm unable to connect at https://jodumont.erpnext.com
with the password I generated with KeepassX (24 caracters, alpha/numerique/symbol ...)

when I ask for a reset it's always said :

```
Invalid Password: Common names and surnames are easy to guess.
Hint: Include symbols, numbers and capital letters in the password
```

Even when I generated a 42 characters password with symbols, numbers and CAPITAL ...
I receive the same message
```
Invalid Password: Common names and surnames are easy to guess.
Hint: Include symbols, numbers and capital letters in the password
```

What is the limit of characters
and do you have any restriction like
need 5CAPITALS LETTERS, 3NUMBERS, 2SYMBOLS ...

Regards!

**Comment by manassolanki** [comment]
I think this has been fixed via https://github.com/frappe/frappe/pull/3536.
If you are frappe cloud customer it will be deployed in your account by the end of the day. Please cross check this once you will be on v8.1.0

**Comment by tundebabzy** [comment]
Hi @judomont
any updates on this issue?

**Comment by madpad** [comment]
Has this issue been fixed? I am on v13.14.1 (version-13) and I still am unable to change passwords...

**Comment by tmark00** [comment]
The problem persist on version 13.18.00
